### DISCUSSION TITLE
Basic App Concept

### DISCUSSION THREAD

**Discussion by Pertempto** [software-factory-collective/mabocados#2] [Design]
## Basic Pitch

Math is a workout for your brain. Daily practice can help you build your brain.

Mabocados is an app full of high-quality mini games which are both engaging and intellectually enriching.

## Goals

- Engaging - The app and games should be a high quality production, full of delightful animations and styling that draw you in. However, we should have limits in place that ensure that the app doesn't become addictive (similar to the once a day strategy of wordle)
- Enriching - Regular use of the app should have a measurable impact on increasing a user's math skills.
- Intuitive - Anyone should be able to pick up the app and get started without much effort. We want to teach math in an intuitive manner. Ideally, almost the entire app could be without text. Math is not bound to any language.
- Open Source - The project will be built on open source technologies, and be open source itself.
- Profitable - Once we have a working prototype, we would like to explore methods for monetizing the project, with the profits being shared proportionally among those who contribute.

## Target Market

These games should be enjoyable to everyone, but ideally they should be very useful for teaching math concepts to elementary school students.

@Pertempto has connections with many home school families who might be interested in this kind of thing

## First Mini Game

A very simple game for practicing multiplication.

You are shown a grid of squares.

You must input the number of squares as quickly as possible.

You have one minute to complete as many grids as possible.

### Ideas
- If the user inputs a value too low, highlight that many squares in red
- If the user inputs a value too large, highlight the whole grid and show some sort of red overflow error
- If the user inputs the correct value, highlight the grid in green

**Comment by Pertempto** [comment]
@JCampana98  does this cover everything important from our initial discussion?

**Reply by JCampana98** [reply]
Yep, I think it covers everything we discussed very well

**Comment by Pertempto** [comment]
@LeanMendez hopefully this can get you up to speed on the basic idea

In general, we're aiming for quality over quantity with this
